RELEASES.md

```diff
@@ -0,0 +1,78 @@
+All the major and minor releases will be made from the master. The tag will be of the format `v<major>.<minor>.<patch>`.
+
+After a minor release, a branch will be created with the format of `release-<major>.<minor>` from the minor tag. Once the branch is ready, all the patches will be released from that branch. 
+For example, once we release `v1.10.0`, a branch will be created as `release-1.10` with the following `v1.10` tag. Next, all future patches will be done against that branch. i.e. `v1.10.0, v1.10.1`.
+
+## Next Release
+
+The next release will be tracked in the milestone under the issue section, with Release tracking. 
+Also, it can be filtered with the label release. If your issue or PR is not present in a milestone, please create a PR to track the next release.
+
+## Coredns Support
+
+The support will be identified with several states:
+
+- __*Active*__: The release is a stable branch that is currently supported and accepting patches.
+- __*Maintenance*__: The release that is currently supported and accepting patches.
+- __*End of Life*__: The release branch is no longer supported, and no new patches will be accepted.
+
+Currently, we are planning to support at least two latest major and minor releases **(N, N-1)**. The latest releases can be any combination of major or minor releases. Examples: a major release followed by two or more minor releases (e.g. 1.9, 1.10). 
+It is unknown when the next release will be made available. So, until then, the current or latest release will be considered the Active release.
+
+For e.g.
+
+The active release is *v1.10 (N)* and *v1.9 (N-1)* is in maintenance. But when the next release rolls out i.e. *v1.11 (N+1)* then *N, N+1* will be the active release, and *N-1* will be in maintenance mode.
+
+Maintenance releases will be supported for upto 6 months or more until the end of life is announced for the branch i.e. v1.9 will be EOL when v1.11 is released, as shown in the diagram.
+These branches will accept bug reports and backports until the end of life. All information will be populated below.
+
+The current state of available support will be:
+
+
+| Release                                                              | Status        | Start         | End of Life             |
+| ---------                                                            | ------------- |---------------|-------------------------|
+| 1.8                                                            | End of Life | Oct 22, 2020  | Jan 14, 2022            |
+| 1.9                                                            | Maintenance | Feb 10, 2022  | Until 1.11 is released. |
+| 1.10                                                            | Active | Sept 19, 2022 | 6 months or more        |
+| 1.11                                                            | Next | TBD           | TBD                     |
```

Comment:
Ideally our support policy should be obvious to the point that this table is not necessary.  I think that the proposal is a combination of both release counts and original release dates makes this too complex.